.circleci/config.yml

```diff
@@ -1,36 +1,26 @@
 # Cache key for CircleCI. We want to invalidate the cache whenever the Bazel workspace or the
 # NPM dependencies changed.
 var_1: &cache_key angular-tsickle-{{ checksum "WORKSPACE" }}-{{ checksum "yarn.lock" }}
-# Default docker image for CircleCI jobs with Bazel installed. The version of Bazel is controlled
-# by the docker image version. e.g "google/bazel:0.22.0" installs Bazel v0.22.0. When updating this
-# version, consider updating the minimum required Bazel version in the "WORKSPACE" file.
-var_2: &default_docker_image l.gcr.io/google/bazel:0.28.0
-
-# Settings common to each job
-var_3: &job_defaults
-  working_directory: ~/tsickle
-  docker:
-  - image: *default_docker_image
-
-# Saves the cache for the current cache key. We store the installed "node_modules" and the Bazel
-# repository cache in order to make subsequent builds faster.
-var_4: &save_cache
-  save_cache:
-    key: *cache_key
-    paths:
-      - "node_modules"
-      - "~/bazel_repository_cache"
 
 version: 2
 
 jobs:
   build:
-    <<: *job_defaults
+    working_directory: ~/tsickle
+    # Default docker image for CircleCI jobs with Bazel installed. The version of Bazel is controlled
+    # by the docker image version. e.g "google/bazel:0.22.0" installs Bazel v0.22.0. When updating this
+    # version, consider updating the minimum required Bazel version in the "WORKSPACE" file.
+    docker:
+      - image: l.gcr.io/google/bazel:0.28.0
     steps:
       - checkout
       - restore_cache:
           key: *cache_key
       - run: bazel run @nodejs//:yarn
-      - run: bazel build ...
-      - run: bazel test ...
-      - *save_cache
+      - run: bazel run @nodejs//:bin/yarn lint
```

Comment:
Just so that I understand - running yarn somehow sets the correct working directory?

Reply:
the working directory is correct either way, but something in the npm-cli.js script caused the problem you saw, I still didn't figure out what - see the comments on your PR

Reply:
Good to know.